src/components/resources/Resources.js

```diff
@@ -10,29 +10,29 @@ export default function Home() {
         Adopting pets is a much more humane way to find your familys newest best
         friend. Every year around 6.5 million companion pets enter shelters in
         just the USA, and thats only dogs and cats. Out of those
-        <b>6.5 million</b>, only <b>3.2 million</b> are adopted and
-        <b>1.5 million</b> are euthanized (Statistics provided by
+        <span> 6.5 million</span>, only <span>3.2 million</span> are adopted and
+        <span> 1.5 million</span> are euthanized (Statistics provided by
         <a
           target="_blank"
           rel="noreferrer"
           href="https://www.aspca.org/animal-homelessness/shelter-intake-and-surrender/pet-statistics"
         >
-          ASPCA
+          {""} ASPCA
         </a>
         ).
-        <br />
-        <b>This leaves 1.8 million pets that go untouched each year.</b>
-        <br />
+        <p>
+          <span>This leaves 1.8 million pets that go untouched each year.</span>
+        </p>
         <p>
           Adopting a pet is cheaper than buying a pet, and you also potentially
           save a life by doing so. Many shelter pets are already house-trained
           and are used to living with humans.{" "}
         </p>{" "}
-        <br />
-        <b>
-          More information about adopting pets can be found at the links below
-        </b>
-        <br />
+        <p>
+          <span>
+            More information about adopting pets can be found at the links below
+          </span>
+        </p>
       </p>
```

Comment:
put this before the p tag at line 23

Reply:
By "this" you mean?

Reply:
closing p tag sorry

Reply:
> put this before the p tag at line 23

If I'm understanding this correctly...
I moved the `<p>` (and its closing tag of course) that's between line 32-34. Here's how it looks like on the webpage.

![SharedScreenshot](https://user-images.githubusercontent.com/80357251/121431237-6e149300-c979-11eb-8560-ae9d4be952f9.jpg)

*"More information about adopting pets can be found at the links below"* is moved from **above the buttons** to **above "This leaves 1.8 million pets that go untouched each year."** paragraph.

Good so far?

Reply:
Yup looks good